packages/utilities/web-vitals/src/index.js

```diff
@@ -100,6 +100,9 @@ const useWebVitals = ({
     const pageExitTime = Date.now();
     const pageAge = pageExitTime - pageLoadTime;
 
+    // Last chance to get the CLS before sending the beacon.
+    getCLS(updateWebVitals, true);
```

Comment:
I always get really confused by how these web-vitals lib functions work, because my understanding is the function will automatically call the `updateWebVitals` callback more than once. So _in theory_, calling the `getCLS` function twice won't make a difference. 🤔 But I'm guessing you've found that it actually does?

> This callback will fire any time either:
>
> * The final value of the metric has been determined.
> * The current metric value needs to be reported right away (due to the page being unloaded or backgrounded).

Reply:
Was just going off of @MattWilliams89 idea to call it twice. Perhaps its not actually needed though